docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -388,12 +388,53 @@ To access the API server from within a Kubernetes cluster, some RBAC related res
 So, when the `kubernetes-client` extension is present, the `kubernetes` extension is going to create those resources automatically, so that application will be granted the `view` role.
 If more roles are required, they will have to be added manually.
 
+=== Deploying to Minikube
+
+https://github.com/kubernetes/minikube[Minikube] is quite popular when a Kubernetes cluster is needed for development purposes. To make the deployment to Minikube
+experience as frictionless as possible, Quarkus provides the `quarkus-minikube` extension. This extension can be added to a project like so:
+
+[source,xml]
+----
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-minikube</artifactId>
+    </dependency>
+----
+
+The purpose of this extension is to generate Kubernetes manifests (`minikube.yaml` and `minikube.json`) that are tailored to Minikube.
+This extension assumes a couple things:
+
+* Users won't be using an image registry and will instead make their container image accessible to the Kubernetes cluster by building it directly
+into Minikube's Docker daemon. To use Minikube's Docker daemon you must first execute:
+
```

Comment:
I think you need a `+` here to make the source part of the bullet point.

Reply:
I didn't know that trick, thanks!